hokusai/production.yml

```diff
@@ -61,6 +61,27 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+        - name: positron-nginx
+          image: artsy/docker-nginx:1.14.2
+          ports:
+            - containerPort: 80
+            - containerPort: 8443
```

Comment:
Not 100% sure on these ports

Reply:
Those look good - will need to update the service spec to match - `80` here https://github.com/artsy/positron/blob/master/hokusai/staging.yml#L116 and `443` here https://github.com/artsy/positron/blob/master/hokusai/staging.yml#L120  (same in `production.yml`)

Reply:
Also to use the `default-websockets` config, Positron itself should be configured to listen on port `8080` rather than `3005` so either set it with `hokusai staging env set PORT=3005` before running `hokusai staging update` or if you want to hardcode it into the container spec you can add this above the `envFrom` directive [here](https://github.com/artsy/positron/blob/master/hokusai/staging.yml#L30) (anything hardcoded will override any values set by `hokusai staging env` in the `positron-environment` ConfigMap
```
env:
  - name: PORT
     value: 8080
```